README.md

```diff
@@ -221,17 +221,28 @@ The body simply contains a byte representation of the HTTP payload that will be
 
 [HDFS organizes](https://hadoop.apache.org/docs/current/hadoop-project-dist/hadoop-hdfs/HdfsDesign.html#The_File_System_Namespace) the data in folders containinig big data files. Such organization is exploited by [`OrionHDFSSink`](doc/design/OrionHDFSSink.md) each time a Flume event is taken from its channel.
 
-Assuming `hdfs_username=myuser`, `service_as_namespace=false` and `attr_persistence=row` as configuration parameters, then the data within the body will be persisted as:
+Assuming `hdfs_username=myuser`, `service_as_namespace=false` and `file_format=json-row` as configuration parameters, then the data within the body will be persisted as:
```

Comment:
An option I was thinking about was to maintain the `attr_persistence` field with values `row` and `column`, and adding the `file_format` parameter with values `row` and `column`. The problem with that approach is that in the future we can find file formats not supporting both row and column modes, but just one of them... or none, e.g. Parquet, which has its own persistence schema.